backend/djangoindia/settings/local.py

```diff
@@ -1,7 +1,9 @@
-from .base import *  # noqa: F403, F401
+import os
 
-DEBUG = True
+from .base import *  # noqa: F403, F401
 
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
 
 ALLOWED_HOSTS = ["*"]
+
+ADMIN_URL = os.environ.get("DJANGO_ADMIN_URL", "admin/")
```

Comment:
Are we thinking of changing `admin url`?

Reply:
lets not expose our admin panel in prod